app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def average_rating(video)
+    avg = video.reviews.inject(0) do |sum,el|
+      sum += el.rating
+    end
+    return "Unrated" if avg ==0
+    (avg.to_f/video.reviews.count).round(2)
```

Comment:
Lots of inconsistent styling here as well. Also you could use `ActiveRecord#average` method. Read more [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average).